internal/cephfs/groupcontrollerserver.go

```diff
@@ -735,3 +735,11 @@ func extractDeleteVolumeGroupError(err error) error {
 
 	return err
 }
+
+// GetVolumeGroupSnapshot return Unimplemented error as its not implemented.
```

Comment:
This should already be inherited from the UnimplementedControllerServer struct that the CSI grpc libs provide.